### ISSUE TITLE
about training mb_melgan with female and male voice dataset.

### ISSUE THREAD

**Issue by TheHonestBob** [TensorSpeech/TensorFlowTTS#707]
I’m here again, thanks for your help. Recently, I use baker and other male voice about 40 min to train fs2 and mb_melgan, I found that the loss of mb_melgan will change suddenly after 5000 steps, from less than 1 to  thousand，I also found that when I use baker and other five speaker dataset which include 3 male and 2 female voice about 30 min ~1 hour for each other， the same thing happened again after 35k steps , shouldn't I use male and female to train mb_melgan and to train fs2? The TTS model doesn't work well which fs2 is trained with baker and male voice and mb_melgan is trained with baker, is it normal?
